src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -159,71 +175,108 @@ const styles = StyleSheet.create({
 });
 
 type VehicleIconProps = {
-  coordinates: Coordinates;
+  vehicle: VehicleWithPosition;
   mode?: AnyMode;
   subMode?: AnySubMode;
   setShouldTrack: React.Dispatch<React.SetStateAction<boolean>>;
+  subscriptionState: SubscriptionState;
+  zoomLevel: number;
 };
 
 const LiveVehicle = ({
-  coordinates,
+  vehicle,
   setShouldTrack,
   mode,
   subMode,
+  subscriptionState,
+  zoomLevel,
 }: VehicleIconProps) => {
+  const {theme} = useTheme();
+
+  const isError =
+    subscriptionState === 'CLOSING' || subscriptionState === 'CLOSED';
+  const isLoading =
+    subscriptionState === 'CONNECTING' || subscriptionState === 'NOT_STARTED';
+  const [isStale, setIsStale] = useState(false);
+
+  useInterval(
+    () => {
+      const secondsSinceUpdate = secondsBetween(
+        vehicle.lastUpdated,
+        new Date(),
+      );
+      setIsStale(STALE_DATA_THRESHOLD_IN_SECONDS < secondsSinceUpdate);
```

Comment:
> Is it really necessary? Isn't it enough with the handling of disconnections, and as long as the connection is open we trust the data?

>  If we wan't the stale data logic I think the threshold should be quite a bit larger than 10s. Maybe 30s?

We've talked a lot about user trust in our data before, and when we release a new data source I **really** want this to be 100% reliable from the start. I think the stale state is a important part of that.

It seems like the data we show here is a lot more reliable than a realtime estimate. From my testing, it seems like it's maybe 1 or 2 seconds old when the app receives it. It is however updated too infrequently, which is why we're working to increase the frequency. I think that after that change, we shouldn't tell the user that "this is good data, the bus is right here" when it's more than 10 or 15 seconds old. 

The bus might show as stale quite a bit in practice, but that should tell the user that when the bus is green, it's actually up to date and reliable data, but while it's gray, the bus might have moved a bit. If I'm late on my way to the bus stop, and I want to know the position of the bus, I don't think 30s old data is good enough.

> When a bus is standing still at a bus stop, do we still get updates on position in the normal interval even though the position is unchanged?

It seems like we do. But we should look into this from more than the few providers I've look at.

Reply:
At least I think the bus icon shouldn't become gray when things are working correctly, as it kinda seems that something is wrong. This is probably more a design question, but I don't think users understand the kinda subtle difference between gray and red icon. 

Maybe settle for 15s now? And also I think the value can be in RemoteConfig.

Reply:
I'll sync up with @ckbilstad tomorrow, since there has been made some changes to the sketches, which might make things more apparent for the user.

I agree that we shouldn't give the user the impression that something is totally wrong when the data is more than 10 or 15 seconds old. But when we expect new data every 5 seconds, I don't think we should tell them that everything is right either.